src/accountsdb/buffer_pool.zig

```diff
@@ -735,190 +757,356 @@ pub const HierarchicalFIFO = struct {
 
 /// slice-like datatype
 /// view over one or more buffers owned by the BufferPool
-pub const CachedRead = struct {
-    buffer_pool: *BufferPool,
-    frame_indices: []const FrameIndex,
-    /// inclusive, the offset into the first frame
-    first_frame_start_offset: FrameOffset,
-    /// exclusive, the offset into the last frame
-    last_frame_end_offset: FrameOffset,
+pub const AccountDataHandle = union(enum) {
+    /// Data owned by BufferPool, returned by .read() - do not construct this yourself (!)
+    cached: CachedRead,
```

Comment:
buffer_pool_read ?

Reply:
renamed in 9d0bf9d3b8a4304181e7f7ed9252f4c4aad847eb